Gemfile.lock

```diff
@@ -266,47 +266,48 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
```

Comment:
### [awesome_print](https://github.com/awesome-print/awesome_print)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.6.1 → 1.9.2

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/awesome_print/1.6.1/1.9.2)_]





<details>
<summary>Commits</summary>

A change of **218** commits. See the full changes on [the compare page](https://github.com/awesome-print/awesome_print/compare/v1.6.1...v1.9.2).

These are the first 10 commits:
- (69815f0) [Updates Documentation](https://github.com/awesome-print/awesome_print/commit/69815f01b2e9d5955ac44959aba2adc2222d2667)
- (526f907) [Improves spec runner harness, switches to better env detecti…](https://github.com/awesome-print/awesome_print/commit/526f90762923f2977b0c4fa134ba57bd8157e762)
- (ee871db) [Updates changelog and bumps version](https://github.com/awesome-print/awesome_print/commit/ee871db0c309be7ddad3a53b5c74b4ab5e56fb97)
- (3f7137c) [Fix Broken specs on Travis](https://github.com/awesome-print/awesome_print/commit/3f7137ce0035d6b433b3e2f6b11f6439d4cac6b0)
- (06a4649) [Update NoBrainer awesome prints](https://github.com/awesome-print/awesome_print/commit/06a4649bb5f349dc7be1413d148f8bddbeec9e5a)
- (13f0278) [Fix specs on Ruby 1.9.3 and Rails 3.2 and 4.2](https://github.com/awesome-print/awesome_print/commit/13f0278b25f7cfc544be11fe1f7d103718cb2d46)
- (de1c1ad) [Travis badge shows the master branch](https://github.com/awesome-print/awesome_print/commit/de1c1ad4fc5cbf50b7575549159653f9668e5936)
- (0d7cef0) [Merge pull request #180 from maurogeorge/fix-spec-travis](https://github.com/awesome-print/awesome_print/commit/0d7cef02ff3f5a6df9129d481f5f695971a12360)
- (7ea8fcc) [Merge pull request #181 from nviennot/master](https://github.com/awesome-print/awesome_print/commit/7ea8fcceefedd3892f4155d8caaad92e82e7148d)
- (4ee2ade) [Be careful about invoking #to_hash on arbitrary objects.](https://github.com/awesome-print/awesome_print/commit/4ee2ade5f72677e2afd92033e6f0d4c244a8e43d)


</details>